high-performing-cache-services.md

```diff
@@ -986,7 +989,7 @@ and expand `coolstore-ui` directory.
 
 You will see javascripts for specific cloud-native services such as cart, catatlog, and order service as above.
 
-Now, we will deploy a presentation layer to OpenShift cluster using `Nodeshift` command line tool, a programmable API that you can use to deploy Node.js projects to `OpenShift`.
+Now, we will deploy a presentation layer to OpenShift cluster using [`Nodeshift`](https://www.npmjs.com/package/nodeshift) command line tool, a programmable API that you can use to deploy Node.js projects to `OpenShift`.
```

Comment:
What linking to `https://nodeshift.dev/`(the project page, not the npm artifact) instead?

Reply:
Ahh I didn't even know that existed! I just googled Nodeshift...

Reply:
Just pushed a commit to change the link